### ISSUE TITLE
v5.2.15 Material Bottom navigation does nothing, always on tab 0

### ISSUE THREAD

**Issue by vallemar** [nativescript-community/ui-material-components#264]
### Which platform(s) does your issue occur on?
- Android (only test)
- device.

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.0.1
### Please, tell us how to recreate the issue in as much detail as possible. 
 I have installed the just released version of https://github.com/nativescript-community/ui-material-components for Tabs and Bottom navigation (v5.2.15) and now my Bottom navigation does nothing. When changing tabs the initial view always remains.

I have returned to v5.2.14 and I still have an error, I have entered package.json of ui-material-bottom-navigation and I see this 
```
...
"dependencies": {
     "@nativescript-community/ui-material-core": "^5.2.15"
  },
```
Even if you install the .14 does it use the .15? if so I confirm that the version is broken in android

**Comment by farfromrefug** [comment]
@vallemar shoot :) I ll fix it sorry

**Comment by vallemar** [comment]
is this line:
```
if (index === -1 || (this._currentFragment && index === this.selectedIndex) || !this._attachedToWindow)
```

**Comment by vallemar** [comment]
For me on Android it is corrected with version v5.2.16. The error https://github.com/nativescript-community/ui-material-components/issues/262 has also been corrected with this version

**Comment by vallemar** [comment]
@farfromrefug  Everything works fine but I have now realized that in the console I have these messages, I do not know if they have been left unintentionally. :)

```

JS: _onAttachedToWindow Error
JS:     at BottomNavigation._onAttachedToWindow (file: src\webpack:\checkpointcurly\node_modules\@nativescript-community\ui-material-bottom-navigation\index.android.js:304:0)
JS:     at Object.AttachListener.onViewAttachedToWindow (file: src\webpack:\checkpointcurly\node_modules\@nativescript-community\ui-material-bottom-navigation\index.android.js:157:0)
JS: setPrimaryItem 0 FragmentBase_vendor_5421_28_TabFragmentImplementation{b7a2b9e} (23c64a6f-ece7-483a-8d6d-836d565be9ad) id=0x12 android:bottomnavigation:18:0}
JS: setPrimaryItem 1 FragmentBase_vendor_5421_28_TabFragmentImplementation{222910} (1aa86bcb-13ba-4daf-a248-2e8d1eb976e6) id=0x12 android:bottomnavigation:18:1}
JS: setPrimaryItem 3 FragmentBase_vendor_5421_28_TabFragmentImplementation{8a02c66} (40a34666-65a3-4962-afee-c5547cac02fc) id=0x12 android:bottomnavigation:18:3}

```

**Comment by farfromrefug** [comment]
@vallemar yep :s will remove the log. so sorry did that too late ...

**Comment by farfromrefug** [comment]
@vallemar logs should be fixed in 5.2.17
